code/style.css

```diff
@@ -6,39 +6,58 @@ body {
   margin: 0;
   padding: 0;
   font-family: 'Montserrat', sans-serif;
-  background: #0026ff;
+  background-image:url(https://images.unsplash.com/photo-1525362081669-2b476bb628c3?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1234&q=80);
+}
+.overlay::before{
+content: "";
+position: absolute;
+top: 0; left: 0;
+width: 100%; height: 100%;
+opacity: 30%;
+background-color: rgb(94, 15, 84);
+}
+
+.overlay-inside {
+  position: relative;
 }
 
 h1 {
   font-weight: bold;
   font-size: 44px;
-  line-height: 34px;
   color: #fff;
   text-align: center;
+  font-family: 'Monoton';
+  margin-bottom: 0;
 }
 
-h2 {
-  font-weight: bold;
-  font-size: 24px;
-  line-height: 34px;
+.chat-with-me {
+  text-align: left;
   color: #fff;
-  text-align: center;
-  margin-bottom: 36px;
+  font-family: 'Covered By Your Grace';
```

Comment:
You are so good with styling with different colors, fonts etc. this contributes a lot to the beat-bot-feeling of the page! inspiring!!